src/ai/backend/manager/data/container_registry/types.py

```diff
@@ -73,3 +73,10 @@ def fields_to_update(self) -> dict[str, Any]:
         self.ssl_verify.update_dict(to_update, "ssl_verify")
         self.extra.update_dict(to_update, "extra")
         return to_update
+
+
+@dataclass
+class ContainerRegistryLocationInfo:
+    project: Optional[str]
+    registry_name: str
+    url: str
```

Comment:
```suggestion
    url: yarl.URL
```
Let's use URL type here